test/support/gravity_mock.ex

```diff
@@ -18,6 +18,15 @@ defmodule GravityMock do
     }
   end
 
+  def get!("/partners/" <> id) do
+    %{
+      body: %{
+        "id" => id,
+        "name" => "Mocked Partner2"
+      }
+    }
+  end
+
```

Comment:
Need to mock the following line: 
https://github.com/artsy/aprd/blob/290d952b20919c2869dc74309d1a32051ecc6dc5/lib/apr/views/seller_slack_view.ex#L6